### DISCUSSION TITLE
How to pass PyInstaller Arguments

### DISCUSSION THREAD

**Discussion by GSTbyMM** [dashingsoft/pyarmor#1995] [Q&A]
While pyarmor gen --pack onefile foo.py packages without doubts but what if one have to pass PyInstaller arguments like --onefile --windowed --noconfirm --noconsole --runtime-tmpdir="C:\gstbymm" --icon=favicon.ico --add-data "static:static" at the same time?

Thanks in Advance.

**Comment by jondy** [comment]
Refer to
http://pyarmor.readthedocs.io/en/latest/topic/repack.html
